scope/utils.py

```diff
@@ -505,30 +505,46 @@ class Dataset(object):
     def __init__(
         self,
         tag: str,
-        path_dataset: str,
+        path_dataset: Union[str, pathlib.Path],
         features: tuple,
         verbose: bool = False,
         **kwargs,
     ):
-        """Load csv file with the dataset containing both data and labels
-        As of 20210317, it is produced by labels*.ipynb - this will likely change in a future PR
+        """Load parquet, hdf5 or csv file with the dataset containing both data and labels
+        As of 20221025, it is produced by scope_download_classification.py
```

Comment:
Is this comment important?

Reply:
Probably not as important now since our usage documentation describes how to download and merge labels and features. I'll go ahead and remove it.